ports/stm32/boards/Passport/modules/ext_settings.py

```diff
@@ -254,6 +254,12 @@ def remove(self, kn):
             self.current.pop(kn, None)
             self.changed()
 
+    def remove_regex(self, pattern):
+        import re
+        matches = [k for k in self.current if re.search(pattern, k)]
```

Comment:
It'd be great if the regex was compiled first so that it isn't parsed each time in `re.search` by using `pattern = re.compile(pattern)` and then `pattern.search(k)`.